lib/rentlinx/models/base.rb

```diff
@@ -51,5 +51,13 @@ def valid_for_post?
         !send(at).nil? && send(at) != ''
       end
     end
+
+    def missing_attributes
+      missing = required_attributes.select do |at|
+        send(at).nil? || send(at) == ''
```

Comment:
It'd be nice to abstract this logic (even though it's simple) since it's now used in three places (the others being <code>valid?</code> and <code>valid_for_post?</code>.